pkg/store/sandbox/sandbox.go

```diff
@@ -36,15 +37,22 @@ type Sandbox struct {
 	Container containerd.Container
 	// CNI network namespace client
 	NetNS *NetNS
+	// StopCh is used to propagate the stop information of the sandbox.
+	container.StopCh
```

Comment:
Let's move stopCh to a separate file under `pkg/store` for now.
Don't want sandbox store and container store import each other.

Reply:
Done.